lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -119,6 +119,45 @@ function createGroup(group, callback) {
     });
 }
 
+/**
+* List all the configGroups created in the IoT Agent.
+*
+* @param {Number} service      Service for wich all the configurations want to be retrieved.
+* @param {Number} limit        Maximum number of entries to return.
+* @param {Number} offset       Number of entries to skip for pagination.
+*/
+function listConfigGroups(service, limit, offset, callback) {
+    const condition = {};
+
+    function toObjectFn(obj) {
+        return obj.toObject();
+    }
+
+    if (service) {
+        condition.service = service;
+    }
+
+    const query = Group.model.find(condition).sort();
+
+    if (limit) {
+        query.limit(parseInt(limit, 10));
+    }
+
+    if (offset) {
+        query.skip(parseInt(offset, 10));
+    }
+
+    async.series([query.exec.bind(query), Group.model.countDocuments.bind(Group.model, condition)], function (
+        error,
+        results
+    ) {
+        callback(error, {
+            count: results[1],
+            configGroups: results[0].map(toObjectFn)
+        });
+    });
+}
+
```

Comment:
Duplicated code, same as `function listGroups(service, limit, offset, callback)`. It is not required.

Reply:
Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1375/commits/11787050de050581b6b76221f20d9c2c671c3b11